docs/user-guide/plugins/github-pull-request-updater.md

```diff
@@ -0,0 +1,58 @@
+# GitHub Pull Request Updater
+
+## Introduction
+The **GitHub Pull Request Updater** plugin automatically adds pipeline status and deployment information as comments on your GitHub pull requests. Every time your Devtron CI pipeline runs, you get instant visibility onto the specific pull request on GitHub. This seamless integration eliminates the need to switch between multiple tools or platforms to track your deployment, Development teams also saves their time by avoiding manual status updates and reducing the back-and-forth communication about deployment states. 
+
+### Prerequisites
+Before integrating the **GitHub Pull Request Updater** plugin, ensure you have an GitHub Personal Access Token (PAT) with appropriate permissions to access the your GitHub repository.
```

Comment:
(Replace 'an' with 'a' in the following line): `have an GitHub Personal Access Token`
(Remove 'the' in the following line): `access the your GitHub repository`

Reply:
done